R/get_source_expressions.R

```diff
@@ -2,10 +2,47 @@
 #'
 #' This object is given as input to each linter
 #' @param filename the file to be parsed.
+#' @return A `list` with three components: (1) `expressions`, a `list` of
```

Comment:
This section is very nice! I think it would be even more useful if the hierarchical structure of the return value is supported by an itemize-ish layout of the documentation.

Maybe nested `\describe{...}`s work well here.

Reply:
Went with `\item` since that's what `?lm` uses, and `\itemize` for the sub-list description. Rendered and it looks passable.